webapp/portlet/src/main/webapp/vue-apps/spaces-overview/components/SpacesOverviewCard.vue

```diff
@@ -38,7 +38,7 @@ export default {
   },
   computed: {
     clickable() {
-      return this.count && this.count !== '-' && 'clickable';
+      return this.count && this.count> 0  && 'clickable';
```

Comment:
Please execute `npm run lint` and ensure that the list of transpilation warnings doesn't increase (in `npm run watch` you have the number of warnings)

```suggestion
      return this.count && this.count > 0  && 'clickable';
```

Reply:
fixed